src/textbooks/textbook-card/TextbookCard.scss

```
@@ -1,9 +1,10 @@
 .textbook-card {
-  padding: $spacer $spacer $spacer map-get($spacers, 4);
+  // stylelint-disable-next-line max-line-length
+  padding: var(--pgn-spacing-spacer-base) var(--pgn-spacing-spacer-base) var(--pgn-spacing-spacer-base) var(--pgn-spacing-spacer-4);
```

[nit]: Lets try something like this
```suggestion
  padding:
      var(--pgn-spacing-spacer-base) var(--pgn-spacing-spacer-base)
      var(--pgn-spacing-spacer-base) var(--pgn-spacing-spacer-4);
```
or
```suggestion
  padding:
      var(--pgn-spacing-spacer-base)
      var(--pgn-spacing-spacer-base)
      var(--pgn-spacing-spacer-base)
      var(--pgn-spacing-spacer-4);
```

---

Changed